index.html

```diff
@@ -5,10 +5,44 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
     <title>Piston-UI</title>
 </head>
 <body>
-  
+   <!-- ------------NORMAL Buttons analogy -->
+   <div>
+    <button class="btn">DEFAULT</button>
+    <button class="btn btn-pri">PRIMARY</button>
+    <button class="btn btn-sec">SECONDARY</button>
+    <button class="btn btn-error">ERROR</button>
+    <button class="btn btn-success">SUCCESS</button>
+</div>
+<!-- ----------Outline buttons analogy -->
+<div style="margin-top: 2rem;">
```

Comment:
It was mentioned to avoid using inline styles, you can create a class name m-top and use it. It follows atomic structure.

Reply:
Thanks a lot Partha for reviewing it.
Actually this is for showcasing purpose for now Its of no use after that just for this review I have kept Because I didnt wanted to include any reviewing CSS in my CSS file